src/marqo/tensor_search/models/index_settings.py

```diff
@@ -57,9 +57,11 @@ class IndexSettings(StrictBaseModel):
     def validate_url_pointer_treatment(cls, values):
         treat_as_images = values.get('treatUrlsAndPointersAsImages')
         treat_as_media = values.get('treatUrlsAndPointersAsMedia')
-
         if treat_as_images is None:
-            treat_as_images = False
+            if treat_as_media is True:
+                treat_as_images = None #False
```

Comment:
treat_as images was None, why do we need to assign it to None again?

Reply:
Because if we default assign it to False, when treat_as_media is True, then we will error out saying ` treat_as_media cannot be False when  treat_as_media is True` even though the user did not set  treat_as_image